lib/lambda.js

```diff
@@ -75,7 +75,12 @@ module.exports = function elasticApmAwsLambda (agent) {
     return function wrappedLambda (payload, context, callback) {
       let parentId
       if (payload.headers !== undefined) {
-        parentId = payload.headers['elastic-apm-traceparent']
+        for (let [key, value] of Object.entries(payload.headers)) {
```

Comment:
`Object.entries` is unfortunately not available in Node.js 6. We have a polyfill you can use instead by requring `object.entries` in the top of the file like so:

```js
const entries = require('object.entries') // TODO: Remove when Node.js 6 is no longer supported

// ...

for (const [key, value] of entries(payload.headers)) {
  // ...
}
```

(I also changed `let` to `const` in the `for` loop as this is going to be a requirement by our linter soon, so we might as well ensure it's fixed already)

Reply:
Thanks for the review! Updated.